src/components/filters.jsx

```diff
@@ -0,0 +1,257 @@
+import React, { useContext } from 'react'
+import { FilterContext } from '../frontend/context/filterContext';
+
+export default function FilterSection() {
+
+    const { products, dispatch, filteredProducts, filteredList } = useContext(FilterContext)
+    console.log(filteredList);
+
+    console.log(filteredProducts);
+    return (
+
+
+        <div id="filters">
+            <div
+                id="filters-btn"
+                className="filter-heading bat-fw-600 bat-flex bat-justify-between bat-align-center"
+            >
+                Filters
+                <span id="cat-icon" className="material-icons">
+                    keyboard_double_arrow_down
+                </span>
+            </div>
+
+            <div className="filters">
+                <div onClick={() => dispatch({ type: "CLEAR_FILTERS" })}
+                    className="filter-heading bat-fw-400 bat-flex bat-justify-end bat-align-center"
+                >
+                    Clear
+                </div>
+                <div className="sorting bat-mar-tb-1 bat-flex bat-flx-dir-col">
+                    <span className="filter-type">PRICE</span>
+
+                    <label className="bat-flex bat-gap-8px bat-align-center" for="sort">
+                        <input type="range" step="25" className="width-100" onChange={(e) => dispatch({ type: "PRICE_RANGE", payload: e.target.value })} list="tickmarks" />
```

Comment:
you should extract this clickHandler as a separate function, the jsx should not have much logic written in it

Reply:
Will make the changes, thank you.